### PR TITLE
#317 - hide version path from swagger

### PR DESCRIPTION
**Pull request by synth**

### CODE REVIEW

**Comment by synth** [comment]
I previously did work under PR #319, but I screwed up the rebase. Hopefully this is better.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ### 0.10.5 (Next)
 
-* Your contribution here.
```
Comment (dblock) [inline CHANGELOG.md]: Put this back please.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+* [#319](https://github.com/ruby-grape/grape-swagger/pull/319): Hide module version from path [@synth](https://github.com/synth)
```
Comment (dblock) [inline CHANGELOG.md]: This is missing a period ;)

**Comment by synth** [comment]
gah, i think I've entered rebase hell again. I rebased squashing the 2nd commit(which fixes the changelog), but then it forced me to pull again causing another merge commit. I don't get it. #ihaterebase

**Comment by dblock** [comment]
Once you have rebased you have to force push your branch to Github, don't pull (save your work first just in case ;)).

**Pull request closed without merging**